exporters/trace/stream/stream.h

```diff
@@ -0,0 +1,180 @@
+#pragma once
+
+#include "opentelemetry/sdk/trace/exporter.h"
+#include "opentelemetry/sdk/trace/span_data.h"
+#include "opentelemetry/nostd/type_traits.h"
+#include "opentelemetry/common/attribute_value.h"
+#include <opentelemetry/version.h>
+
+#include <sstream>
+#include <iostream>
+#include <map>
+
+namespace nostd    = opentelemetry::nostd;
+namespace sdktrace = opentelemetry::sdk::trace;
+
+// AttributeType to help with printing attributes
+enum AttributeType
+{
+  TYPE_BOOL,
+  TYPE_INT,
+  TYPE_INT64,
+  TYPE_UINT,
+  TYPE_UINT64,
+  TYPE_DOUBLE,
+  TYPE_STRING,
+  TYPE_SPAN_BOOL,
+  TYPE_SPAN_INT,
+  TYPE_SPAN_INT64,
+  TYPE_SPAN_UINT,
+  TYPE_SPAN_UINT64,
+  TYPE_SPAN_DOUBLE,
+  TYPE_SPAN_STRING
+};
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace exporter
+{
+namespace trace
+{
+class StreamSpanExporter final : public sdktrace::SpanExporter
+{
+public:
+    explicit StreamSpanExporter(std::ostream &sout = std::cout,
+                                bool isShutdown = false) noexcept;
```

Comment:
Can we add a doc string to the class and the constructor? I'm not clear about what the intention of the `isShutdown` argument is.

Reply:
Yeah I'll add a doc string. I will remove isShutdown from the constructor since it should always be false when the exporter is created